BIMServer.py

```diff
@@ -80,7 +80,7 @@ def __init__(self):
         self.prefs = FreeCAD.ParamGet("User parameter:BaseApp/Preferences/Mod/Arch")
         self.Projects = []
         self.Revisions = []
-        self.RootObjects = Draft.getObjectsOfType(FreeCAD.ActiveDocument.Objects,"Site")+Draft.getObjectsOfType(FreeCAD.ActiveDocument.Objects,"Building")
+        self.RootObjects = Draft.getObjectsOfType(FreeCAD.ActiveDocument.Objects,"IfcSite")+Draft.getObjectsOfType(FreeCAD.ActiveDocument.Objects,"IfcBuilding")
```

Comment:
This indeed comes from older Arch types. Since there might still be some old files around, I would suggest extending instead of replacing:

```suggestion
        self.RootObjects += Draft.getObjectsOfType(FreeCAD.ActiveDocument.Objects,"IfcSite")+Draft.getObjectsOfType(FreeCAD.ActiveDocument.Objects,"IfcBuilding")
```

Reply:
Yes true. My change would break a lot of old setups. Made changes accordingly. Thanks for the review!